### ISSUE TITLE
Icon request for Astro

### ISSUE THREAD

**Issue by RickyC0626** [pandier/intellij-discord-rp#90]
**File extension:** `.astro`

https://astro.build/press/

Logomark on dark
![image](https://github.com/user-attachments/assets/2e2c6997-a36f-4278-a7bc-0345d2dfec09)

**Comment by pandier** [comment]
Relatively straightforward, proposal:

![astro@ 5x](https://github.com/user-attachments/assets/fb405aa4-8a95-482a-9669-eb4c810d2538)

**Comment by RickyC0626** [comment]
Perhaps darker leaning toward violet? Similar to the background gradient on Astro's main page

https://astro.build/
![image](https://github.com/user-attachments/assets/dd01f5a5-e5d6-479b-870a-901c73729ec1)

**Comment by Cleboost** [comment]
I agree for a darker purple. As for the gradient, it seems to me that no other icon has one and that Pandier will not - want it. I think just in dark mlis would be enough

**Comment by pandier** [comment]
Here's a more purple color:

![astro@ 5x](https://github.com/user-attachments/assets/2d09ec5a-8bc1-4446-abb3-e23820677b3e)

As for the dark color and white logo: I don't really want to do that, because it would break the consistency. All icons currently use a vibrant color with a dark logo

**Comment by Cleboost** [comment]
I prefer second
